pipestat_example/.looper.yaml

```diff
@@ -0,0 +1,9 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+pipestat:
+  sample:
```

Comment:
I'm slightly confused by this nomenclature; what is `pipestat.sample`, exactly? Is `sample_pipeline` maybe more clear?

Reply:
Reverted to using a config file:
`pipestat_config: ./global_pipestat_config.yaml`